### ISSUE TITLE
How are you using react-photo-gallery?

### ISSUE THREAD

**Issue by neptunian** [neptunian/react-photo-gallery#129]
I'd love to see some examples of react-photo-gallery in the wild!  I'd like to have a samples section, so let me know if you'd like to have your site linked to in the README.

**Comment by docloulou** [comment]
https://lucas.zerbib.xyz/ ;)

**Comment by bushblade** [comment]
This is an awesome library I love it and have been asked a few times how I made my gallery, happy to point them to this repo. [My Gallery](https://bushblade.co.uk/woodlore-clone)

**Comment by wmsiddiqui** [comment]
I'm using a version which I forked (and recently opened a PR for against this repo). My images are hosted on my google drive and are taking a while to load. So I've added a spinner, and had a few additional ideas, like transitions. Your library is very nice and easy to use already and I didn't want to re-invent the wheel. So maybe if I get around to it I'll make a separate library which uses yours as a dependency. Let me know how you'd feel about that :)

https://wmsiddiqui.com/gallery
